ios/Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
```

Comment:
Now that we removed Gemfile in the root, can we include the below in this file ?

ruby '3.1.2'

gem 'cocoapods', '1.14.3'

Reply:
Thanks a lot!